cpp/CMakeLists.txt

```diff
@@ -64,6 +64,8 @@ option(BUILD_CUML_STD_COMMS "Build the standard NCCL+UCX Communicator" ON)
 
 option(BUILD_CUML_MPI_COMMS "Build the MPI+NCCL Communicator (used for testing)" OFF)
 
+option(BUILD_FAISS "Build the FAISS library for nearest neighbors search on GPU" OFF)
```

Comment:
I wonder if `BUILD_STATIC_FAISS` would be a better name, just for added clarity? Either way, could you add documentation of this option to https://github.com/rapidsai/cuml/blob/branch-0.15/cpp/README.md#building-cuml ?

Reply:
The other thing is I think it might be a good idea to add a flag to turn building `faiss` statically in the base `build.sh`, to ease things for people that look to build things outside of conda. It could be in this PR or a follow up with  gh issue

Reply:
Done.